components/esp_netif/lwip/esp_netif_lwip_slip.c

```diff
@@ -162,7 +162,13 @@ void esp_netif_lwip_slip_input(void *h, void *buffer, unsigned int len, void *eb
     ESP_LOG_BUFFER_HEXDUMP(TAG, buffer, len, ESP_LOG_DEBUG);
 
     // Update slip netif with data
-    slipif_received_bytes(netif->lwip_netif, buffer, len);
+    const int max_batch = 255;
+    int sent = 0;
+    while(sent < len) {
+        int batch = len > max_batch ? max_batch : len;
```

Comment:
I think we have to compare with the *remaining length* here rather than the total `len` itself, since it would always be greater than... Sorry must have confused you in the lwip PR.

Reply:
Yep, good catch. Fixed.